Code/Source/Entity/EntityManager.h

```diff
@@ -14,42 +14,37 @@
  */
 #pragma once
 
-#include <AtomLyIntegration/CommonFeatures/Mesh/MeshComponentBus.h>
 #include <AzCore/Component/EntityBus.h>
 #include <AzCore/Component/EntityId.h>
-#include <rgl/api/core.h>
+#include <AzCore/std/containers/vector.h>
+
+struct Entity;
```

Comment:
I think that using RGL types is more informative about what is going on. It indicates that it is a handler to RGL (not to the game engine or whatever).